monai/transforms/transform.py

```diff
@@ -254,17 +278,22 @@ class LazyTransform(Transform, LazyTrait):
     dictionary transforms to simplify implementation of new lazy transforms.
     """
 
-    _lazy_evaluation: bool = False
+    def __init__(self, lazy: bool | None = False):
```

Comment:
this constructor is introduced by this PR, all the subclasses should have `super().__init__` calls to ensure this is executed during initialisation.

Reply:
Done